guides/doc-Upgrading_and_Updating/topics/cloning_satellite_server.adoc

```diff
@@ -196,7 +196,7 @@ After you install the `satellite-clone` tool, you can adjust any configuration t
 . Click the *Refresh* button, then click *Close* to return to the list of subscriptions.
 . Verify that the available subscriptions are correct.
 . Follow the instructions in the `/usr/share/satellite-clone/logs/reassociate_capsules.txt` file to restore the associations between {SmartProxies} and their lifecycle environments.
-. Update your network configuration, for example, DNS, to match the target server’s IP address with its new host name. The `satellite-clone` tool changes the hostname to the source server's hostname. If you want to change the hostname to something different, you can use the `satellite-change-hostname` tool. For more information, see {AdministeringDocURL}sect-Red_Hat_Satellite-Administering_Red_Hat_Satellite-Renaming_a_Server[Renaming a {Project} or {SmartProxyServer}] in _Administrating {ProjectName}_.
-. If the source server uses the `virt-who` daemon, install and configure it on the target server. Copy all the `virt-who` configuration files in the `/etc/virt-who.d/` directory from the source server to the same directory on the target server.  For more information, see https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProductVersion}/html/configuring_virtual_machine_subscriptions_in_red_hat_satellite/index[_Configuring Virtual Machine Subscriptions_].
+. Update your network configuration, for example, DNS, to match the target server’s IP address with its new host name. The `satellite-clone` tool changes the hostname to the source server's hostname. If you want to change the hostname to something different, you can use the `satellite-change-hostname` tool. For more information, see https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProjectVersion}/html-single/administering_red_hat_satellite/index#sect-{Project_Link}-Administering_{Project_Link}-Renaming_a_Server[Renaming a {Project} or {SmartProxyServer}] in _Administrating {ProjectName}_.
+. If the source server uses the `virt-who` daemon, install and configure it on the target server. Copy all the `virt-who` configuration files in the `/etc/virt-who.d/` directory from the source server to the same directory on the target server.  For more information, see {BaseURL}configuring_virtual_machine_subscriptions_in_red_hat_satellite/index[_Configuring Virtual Machine Subscriptions in {ProjectName}_].
 
```

Comment:
This will fail the link validator as we don't have an upstream version of this link and the BASEURL will cause the linkchecker to validate the docs.theforeman.org/configuring_virtual_machine_subscriptions_in_red_hat_satellite/index.html etc etc
I removed the baseurl for that purpose.